src/xdocs/config_metrics.xml

```diff
@@ -216,6 +222,21 @@
     &lt;property name=&quot;max&quot; value=&quot;5&quot;/&gt;
 &lt;/module&gt;
         </source>
+
+        <p>
+          Override property <code>overridedPackages</code> to mark some packages as excluded.
+
+          Note, that `excludedPackages` will not exclude classes, imported via wildcard
```

Comment:
We should also note that classes in the same package can't be excluded either as there is no `import` statement and Checkstyle can't be sure it is from same package or from `java.lang`.